### PR TITLE
Add curl.bank.engineering

### PR DESCRIPTION
**Pull request by reesericci**
Hello!

I made a project for Bank: Hack Club BancURL. It fetches from the transparency API the latest balance & transaction data and displays them when you cURL the server!  I was wondering if I could ship it on https://curl.bank.engineering. I doubt it - but I thought it was worth a shot.

You can try it by running `curl https://hcb.deno.dev/${orgname}`

Screenshot:

image

### CODE REVIEW

**Comment by reesericci** [comment]
Also all the source code is available at https://dash.deno.com/playground/hcb

**Comment by reesericci** [comment]
Did that fix it?

File: bank.engineering.yaml [hunk 1]
```diff
@@ -3,3 +3,16 @@ changelog:
...
+  type: CNAME
+  value: c82b414cd89d32fd90685ead._acme.deno.dev.
+curl:
+  ttl: 1
+  type: A
+  value: 34.120.54.55
+curl:
+  ttl: 1
+  type: AAAA
+  value: 2600:1901:0:6d85::
```
Comment (cjdenio) [inline bank.engineering.yaml:17]: Wrap this in quotes since it contains colons

Comment (cjdenio) [inline bank.engineering.yaml:17]:
```suggestion
curl:
  - ttl: 1
    type: A
    value: 34.120.54.55
  - ttl: 1
    type: AAAA
    value: 2600:1901:0:6d85::

```

Comment (reesericci) [reply bank.engineering.yaml:17]: ok

Comment (reesericci) [reply bank.engineering.yaml]: so the mapping was fine - i thought that's what the linter was complaining about.

Comment (reesericci) [reply bank.engineering.yaml]: my dumb arse commited before merging your changes -yay

Comment (cjdenio) [reply bank.engineering.yaml]: @reesericci you also have to nest both records under the same `curl` key

**Comment by reesericci** [comment]
how do I fix that @cjdenio

**Review by cjdenio — APPROVED** [review]

**Pull request merged**